client/state/reader/streams/reducer.js

```diff
@@ -133,7 +136,7 @@ export const pendingItems = ( state = PENDING_ITEMS_DEFAULT, action ) => {
 			}
 
 			maxDate = moment( streamItems[ 0 ].date );
-			minDate = moment( last( streamItems ).date );
+			minDate = moment( streamItems[ streamItems.length - 1 ].date );
```

Comment:
Looks safe, above there is a check that ensures `streamItems` has at least 1 element.